gap/ordered.gd

```diff
@@ -99,7 +99,7 @@ BindGlobal( "OrderedSetDSFamily", NewFamily("OrderedSetDSFamily") );
 #! object will have.<P/>
 #! The optional argument <A>lessThan</A> must be a binary function that returns <K>true</K> if
 #! its first argument is less than its second argument, and <K>false</K> otherwise. The default
-#! <A>lessThan</A> is &GAP;'s built in &leq;.<P/>
+#! <A>lessThan</A> is &GAP;'s built in $\leq$.<P/>
```

Comment:
Actually, the original text doesn't even make sense, as `\leq` is of course "less than or equal". Urgh. It really should reference the GAP operation `\<` resp. `LT`, set as code (`<C>...`) but of course the `<` needs escaping in XML... 

But I am not asking you to do that (though of course I wouldn't mind if you did ;-), just pointing out that this is fixing text that's wrong to start with *sigh*

Reply:
Is the XML entity &lt; defined?  I will check.  Either way, I'll try to fix it.

Reply:
Yes, `&lt;` is defined!

Reply:
Yes, and it produces the right output for text and HTML both.  See the new commit.